src/assets/css/main.css

```diff
@@ -213,6 +213,13 @@ blockquote {
   overflow-y: auto;
 }
 
+.comment-node > div > div > .md-div p {
```

Comment:
Please use bootstrap classes on the elements directly where possible.

Reply:
Ah alright thanks for the feedback, I'll look into it

Reply:
I can see in comment-node.tsx that many of the classes are as specific as individual rules, but there is no rule just for flex, so would the addition of the following be acceptable:

```css
.flex {
  display: flex !important;
}
```

The \<p> element doesn't have flex, so it would need this class. The other two rules there are already classes for

Reply:
I just realized that this is for HTML generated from markdown, so my original suggestion isn't too great. Your original solution is closer than your revised solution. Can you get it to work with `.comment-node .md-div p`? If I'm not mistaken, selecting for multiple levels of direct child divs is unnecessary.

Reply:
Yeah that should work just fine, but I'll have to see if there are other .md-div's that aren't the one containing the markdown text, otherwise it could screw with the formatting of other things there too

Reply:
Okay yeah that didn't take long haha, yeah there's only one use of the class .md-div in comment-node.tsx, so that should work just fine

The next issue is where it should be placed, in main.css, or in every individual theme scss?